nativelib/src/main/resources/types.h

```diff
@@ -1,18 +1,28 @@
 #ifndef __TYPES_H
 #define __TYPES_H
 
-typedef unsigned long scalanative_dev_t;
-typedef unsigned int scalanative_mode_t;
-typedef unsigned long long scalanative_ino_t;
-typedef unsigned int scalanative_uid_t;
-typedef unsigned int scalanative_gid_t;
-typedef long long scalanative_off_t;
-typedef long int scalanative_time_t;
-typedef long long scalanative_blkcnt_t;
-typedef long scalanative_blksize_t;
-typedef unsigned long scalanative_nlink_t;
+// this is cross-patform definition of integer types
+typedef signed char scalanative_int8_t;
+typedef short scalanative_int16_t;
+typedef int scalanative_int32_t;
+typedef long long scalanative_int64_t;
+typedef unsigned char scalanative_uint8_t;
+typedef unsigned short scalanative_uint16_t;
+typedef unsigned int scalanative_uint32_t;
+typedef unsigned long long scalanative_uint64_t;
```

Comment:
inttypes.h is part of Windows 10 SDK starts from Visual Studio 2013, but it could be installed on windows 7 (there is special steps for it)
There is also a header on windows: <cinttypes> which is very similar, I just need to investigate this case more and make it support older windows versions.